Sources/Testing/Running/Configuration.swift

```diff
@@ -137,38 +137,16 @@ public struct Configuration: Sendable {
 
   // MARK: - Test selection
 
-  /// The selected tests to run, if any.
-  ///
-  /// This property should be used for testing membership (whether a test ID has
-  /// been selected) since it is more optimized for that use case. It also
-  /// provides the backing storage for ``selectedTestIDs``.
-  ///
-  /// This property is optional and defaults to `nil` because it is possible to
-  /// select specific tests to run but not provide any tests in that list. That
-  /// is a supported use case: it results in zero tests being run and no issues
-  /// recorded.
-  ///
-  /// A practical example of when this situation can happen is when testing is
-  /// configured via an Xcode Test Plan, the "Automatically Include New Tests"
-  /// option is disabled, and zero tests are enabled.
-  var selectedTests: Test.ID.Selection?
-
-  /// The IDs of the selected tests to run, if any.
-  ///
-  /// This property is optional and defaults to `nil` because it is possible to
-  /// select specific tests to run but not provide any tests in that list. That
-  /// is a supported use case: it results in zero tests being run and no issues
-  /// recorded.
-  ///
-  /// A practical example of when this situation can happen is when testing is
-  /// configured via an Xcode Test Plan, the "Automatically Include New Tests"
-  /// option is disabled, and zero tests are enabled.
-  public var selectedTestIDs: Set<Test.ID>? {
-    get {
-      selectedTests?.testIDs
-    }
-    set {
-      selectedTests = newValue.map { .init(testIDs: $0) }
-    }
-  }
+  // TODO: Write DocC documenting this typealias.
+  //
+  // (Note: this typealias may also be used to eventually implement a similar
+  // filter property for **skipped** tests. But that doesn't need to be
+  // mentioned in the DocC.)
+  public typealias TestPredicate = @Sendable (Test) -> Bool
+
+  // TODO: Choose better name for this property.
+  //
+  // TODO: Write DocC documenting this property, explaining its purpose and
+  // potentially including an example of its most common usage patterns.
+  public var testSelectionFilter: TestPredicate?
```

Comment:
This name seems off to me. Maybe just `testFilter`?

Reply:
I'm trying to choose a name that will also allow us to eventually add a corresponding property to filter _out_ excluded or skipped tests. Conceptually, that would act as a "subtract from set" operation, subtracting from the _selected_ tests, so that property could eventually be named something like `testSkipFilter`.

I am definitely open to better naming suggestions still, but that's why I thought it best to denote what _kind_ of filter this is: include vs. exclude.

Reply:
I'd say it's idiomatic in the stdlib that "filter", i.e. `.filter { ... }`, is inclusive, FWIW.

Reply:
I updated it to filter for now, it is true that the sodlib is inclusive: https://developer.apple.com/documentation/swift/string/filter(_:)